integration-tests/bats/helper/local-remote.bash

```diff
@@ -134,6 +134,7 @@ SKIP_SERVER_TESTS=$(cat <<-EOM
 ~sql-checkout.bats~
 ~cli-hosted.bats~
 ~profile.bats~
+~ls.bats~
```

Comment:
Why skip your shiny new tests? Do they not work with the server running?

Reply:
they use `checkout` :(